server/core/config/valid/global_cfg.go

```diff
@@ -186,62 +160,71 @@ func NewGlobalCfgFromArgs(args GlobalCfgArgs) GlobalCfg {
 		applyReqs = append(applyReqs, PoliciesPassedApplyReq)
 	}
 
-	allowCustomWorkflows := false
-	deleteSourceBranchOnMerge := false
+	var deleteSourceBranchOnMerge, allowCustomWorkflows bool
+	allowedOverrides := []string{}
 	if args.AllowRepoCfg {
 		allowedOverrides = []string{ApplyRequirementsKey, WorkflowKey, DeleteSourceBranchOnMergeKey}
 		allowCustomWorkflows = true
 	}
 
-	return GlobalCfg{
-		Repos: []Repo{
-			{
-				IDRegex:                   regexp.MustCompile(".*"),
-				BranchRegex:               regexp.MustCompile(".*"),
-				ApplyRequirements:         applyReqs,
-				PreWorkflowHooks:          args.PreWorkflowHooks,
-				Workflow:                  &defaultWorkflow,
-				AllowedWorkflows:          allowedWorkflows,
-				AllowedOverrides:          allowedOverrides,
-				AllowCustomWorkflows:      &allowCustomWorkflows,
-				DeleteSourceBranchOnMerge: &deleteSourceBranchOnMerge,
-			},
-		},
+	repo := Repo{
+		IDRegex:                   regexp.MustCompile(".*"),
+		BranchRegex:               regexp.MustCompile(".*"),
+		ApplyRequirements:         applyReqs,
+		PreWorkflowHooks:          args.PreWorkflowHooks,
+		Workflow:                  &defaultWorkflow,
+		AllowedWorkflows:          []string{},
```

Comment:
Yeah, this part should be covered further down. It only is populated if platform mode is enabled, otherwise the values in this field shouldn't matter.